### PR TITLE
Up to Production v1.7

### PR DESCRIPTION
**Pull request by rindibudiaramdhan**
## Overview
- Fix Console Command VerificationStatusGenerator
  - Revision value for `status_rank` & `verification_status` when `status` is `rejected`
- Fix API Update Vaccine Request
  - Setup Commit & Rollback DB Transaction
- **[NEW!]** API Check Stock by Vaccine Request ID
- **[NEW!]** API Check Stock by Material ID
- **[NEW!]** API Vaccine Tracking
- **[NEW!]** API Vaccine Tracking Detail
- **[NEW!]** API Vaccine Tracking Products
- Fix Ticketing Bug [T105](https://app.clickup.com/t/2jtxz1j)
- Fix Ticketing Bug [T116](https://app.clickup.com/t/2na0g1p)
- Fix Ticketing Bug [T122](https://app.clickup.com/t/2nv6xhr)
- Fix Ticketing Bug [T123](https://app.clickup.com/t/2p6pr3y)
- Fix CICD API Check Stock
- Fix Codeclimate API Check Stock by Material ID

## Evidence
- title: Up to Production v1.7
- project: Pikobar Logistik
- participants: @rindibudiaramdhan @ayocodingit

### CODE REVIEW

File: app/VaccineWmsJabar.php [hunk 1]
```diff
@@ -118,7 +118,7 @@ static function sendVaccineRequest(VaccineRequest $vaccineRequest)
 
             // Pre-Validating before Integrating to WMS Poslog
             $isValidToIntegrate = self::isValidToIntegrate($items);
-            if (!$isValidToIntegrate['status'] == Response::HTTP_OK) {
+            if (!($isValidToIntegrate['status'] == Response::HTTP_OK)) {
```
Comment (ayocodingit) [inline app/VaccineWmsJabar.php:121]: kang @rindibudiaramdhan `$isValidToIntegrate['status']` kalau ini teh data nya boolean kan yah, kenapa di compare dengan `http_ok` yah kang

Comment (rindibudiaramdhan) [reply app/VaccineWmsJabar.php:121]: oh tidak `boolean`, kang @ayocodingit 
Variabel `$isValidToIntegrate` isinya array yang berasal dari function `isValidToIntegrate($items)`.
Kira-kira apakah membingungkan ya penamaan variabelnya? 😄

Comment (ayocodingit) [reply app/VaccineWmsJabar.php:121]: mungkin `$dataToIntegrate` atau `$payloadToIntegrate` atau `$itemToIntegrate` 😕

Comment (rindibudiaramdhan) [reply app/VaccineWmsJabar.php:121]: sip saya ganti ya kesini (`$dataToIntegrate`)👍

Comment (rindibudiaramdhan) [reply app/VaccineWmsJabar.php]: sudah di update disini ya [15cc2e3](https://github.com/jabardigitalservice/pikobar-logistik-api/pull/448/commits/15cc2e3a4e450cd6d9c22cb79f5e78310f789053) kang @ayocodingit

**Review by ayocodingit — APPROVED** [review]

**Pull request merged**